client/remoterepo.c

```diff
@@ -540,7 +540,7 @@ TDNFGetFileHashFromMetalink(
 }
 
 uint32_t
-TDNFAppendResourceURlToList(
+TDNFAllocateResourceURL(
     TDNF_METALINK_URLS **metalink_tail,
```

Comment:
This just `metalink_url` please remove `_tail`.

Reply:
Yeah, missed to change this. Thanks! modified it now.